maintainers/maintainers-list: add Matrix IDs

rnhmjoj opened the pull request:
###### Motivation for this change

- Quickly find a maintainer's contact
- Link Matrix identities to Github and email

###### To do

- [x] evaluate the list
- [x] verify all IDs

~I've scraped some of the Nix/NixOS Matrix rooms and tried to match user
names, avatars, user IDs etc. with other information from this list.
Note that this is all guesswork, the identities are not verified in any
way, yet.~

~Please, take a look at the diff and tell me if your Matrix ID is
correct and you're ok with adding it the list. Thank you.~

I think you're active on the NixOS Matrix space, so would you like to have your Matrix user ID included in the maintainers list? If so, please leave a comment and I'll be sure to add it.
Thank you.

@AluisioASG @Baughn @CRTified @GuillaumeDesforges @SuperSandro2000
@0x4A6F @aaronjanse @abathur @adisbladis @ajs124 @AmineChikhaoui @andir
@applePrincess @arianvp @shados @artturin @asbachb @bachp @bb010g
@bendlas @berberman @blaggacao @blitz @buckley310 @cdepillabout
@cheriimoya @chvp @cleverca22 @codyopel @cole-h @colemickens @cyplo
@dadada @dali99 @dasJ @davidak @davidtwco @delroth @domenkozar
@dotlambda @drupol @dywedir @edude03 @edwtjo @edolstra @ekleog @eliasp
@ElvishJerricco @emilazy @emmanuelrosa @endocrimes @ereslibre
@ericson2314 @etu @evalexpr @expipiplus1 @eyJhb @fabaff @cideM @felschr
@fgaz @figsoda @fortuneteller2k @fridh @frogamic @fzakaria @ggpeti
@gilligan @globin @cillianderoiste @grahamc @gytis-ivaskevicius
@happysalada @hax404 @mweinelt @higebu @hmenke @hoverbear @hqurve @htr
@iammrinal0 @Icy-Thought @immae @infinisil @IreneKnapp @jfroche
@06kellyjac @joepie91 @jojosch @jonringer @jonas-schievink @jtojnar
@kalbasit @kevincox @kirelagin @kloenk @kraem @L-as @L3afMe @laikq
@Lassulus @ldesgoui @legendofmiracles @nlewo @lheckemann @lovesegfault
@lschuermann @lukegb @lunik1 @ma27 @manveru @maralorn @mattchrist
@matthiasbeyer @maxeaubrey @mic92 @midchildan @misuzu @mjlbach @mkg20001
@mlieberman85 @mlvzk @mudrii @musfay @mvnetbiz @ncfavier @nh2
@ninjatrappeur @nixinator @noneucat @nrdxp @numkem @obfusk @onny
@Pacman99 @petabyteboy @piegamesde @nbp @plabadens @polygon @polykernel
@primeos @princemachiavelli @prusnak @psibi @pstn @putchar @alyssais
@raboof @ralith @rissson @Rizary @rmcgibbo @roberth @roosemberth @ryantm
@s1341 @samueldr @sdier @stigtsp @siraben @solson @spacekookie
@spencerjanssen @srhb @stephank @superherointj @symphorien @Synthetica9
@tadfisher @taktoa @talyz @teto @tfc @thefloweringash @ThibautMarty
@tilpner @tnias @tomberek @toonn @unode @vaibhavsagar @valodim @vcunat
@Xe @xfix @yayayayaka @yorickvp @yu-re-ka @yusdacra @zakame @zhaofengli

rnhmjoj committed:
maintainers/maintainers-list: add Matrix IDs

drupol commented:
Good for me

CRTified commented:
Correct ID and fine for me.

blitz commented on maintainers/maintainer-list.nix:
```diff
@@ -1430,6 +1449,7 @@
   };
   blitz = {
     email = "js@alien8.de";
+    matrix = "@js:ukvly.org";
```
This is correct.

emmanuelrosa commented on maintainers/maintainer-list.nix:
```diff
@@ -3251,6 +3299,7 @@
   };
   emmanuelrosa = {
     email = "emmanuel_rosa@aol.com";
+    matrix = "@emmanuelrosa:matrix.org";
```
This is correct.

chvp commented on maintainers/maintainer-list.nix:
```diff
@@ -2011,6 +2034,7 @@
   };
   chvp = {
     email = "nixpkgs@cvpetegem.be";
+    matrix = "@charlotte:cvpetegem.be";
```
```suggestion
    matrix = "@charlotte:vanpetegem.me";
```

rnhmjoj replied:
Thank you, fixed.

eliasp commented on maintainers/maintainer-list.nix:
```diff
@@ -3191,6 +3237,7 @@
   };
   eliasp = {
     email = "mail@eliasprobst.eu";
+    matrix = "@eliasp:kde.org";
```
:+1:

SuperSandro2000 commented on maintainers/maintainer-list.nix:
```diff
@@ -10591,11 +10754,13 @@
   };
   SuperSandro2000 = {
     email = "sandro.jaeckel@gmail.com";
+    matrix = "@sandro:supersandro.de";
```
👍🏼

jojosch commented on maintainers/maintainer-list.nix:
```diff
@@ -5368,6 +5453,7 @@
   jojosch = {
     name = "Johannes Schleifenbaum";
     email = "johannes@js-webcoding.de";
+    matrix = "@jojosch:jswc.de";
```
:+1:

ajs124 reviewed (changes requested):
I'm missing a `.`. Should be `@andreas.schraegle:helsinki-systems.de`.

rnhmjoj commented:
@ajs124 Fixed, thank you.

dasJ commented:
Also from our Matrix server:
```
Scriptkiddi = "@fritz.otlinghaus:helsinki-systems.de";
SlothOfAnarchy = "@michel.weitbrecht:helsinki-systems.de";
```

buckley310 commented:
Mine is good.

davidak commented:
Correct ID and fine to add.

Great initiative, thanks! :sparkles:

rnhmjoj commented:
> Also from our Matrix server:

Thank you! I added both.

> Also, that should be documented in the comment at the beginning of the file so new users know to add the info.

Done.

Artturin commented on maintainers/maintainer-list.nix:
```diff
@@ -1108,6 +1122,7 @@
   };
   artturin = {
     email = "artturin@artturin.com";
+    matrix = "artturin:matrix.org";
```
```suggestion
    matrix = "@artturin:matrix.org";
```

rnhmjoj replied:
Fixed!

D4ndellion reviewed (approved):
Good for my part

aaronjanse commented on maintainers/maintainer-list.nix:
```diff
@@ -124,6 +127,7 @@
   };
   aaronjanse = {
     email = "aaron@ajanse.me";
+    matrix = "@aaronajanse:matrix.org";
```
This should be @aaronjanse:matrix.org

rnhmjoj replied:
Fixed, thank you.

figsoda commented on maintainers/maintainer-list.nix:
```diff
@@ -3657,12 +3717,14 @@
   };
   figsoda = {
     email = "figsoda@pm.me";
+    matrix = "@figsoda:matrix.org";
```
👍

figsoda commented:
Not everybody was notified, I think github is limiting the amount of mentions

rnhmjoj commented:
> Not everybody was notified, I think github is limiting the amount of mentions

Yes, looks like it. I'll try to break it into several comments.

rnhmjoj commented:
Ping:
@endocrimes @ereslibre @ericson2314 @etu @evalexpr @expipiplus1 @eyJhb @fabaff
@cideM @felschr @fgaz @figsoda @fortuneteller2k @fridh @frogamic @fzakaria
@ggpeti @gilligan @globin @cillianderoiste @grahamc @gytis-ivaskevicius
@happysalada @hax404 @mweinelt @higebu @hmenke @hoverbear @hqurve @htr
@iammrinal0 @Icy-Thought @immae @infinisil @IreneKnapp @jfroche @06kellyjac
@joepie91 @jojosch @jonringer @jonas-schievink @jtojnar @kalbasit @kevincox
@kirelagin @kloenk @kraem @L-as @L3afMe @laikq

rnhmjoj commented:
Ping:
@Lassulus @ldesgoui @legendofmiracles @nlewo @lheckemann @lovesegfault
@lschuermann @lukegb @lunik1 @ma27 @manveru @maralorn @mattchrist
@matthiasbeyer @maxeaubrey @mic92 @midchildan @misuzu @mjlbach @mkg20001
@mlieberman85 @mlvzk @mudrii @musfay @mvnetbiz @ncfavier @nh2 @ninjatrappeur
@nixinator @noneucat @nrdxp @numkem @obfusk @onny @Pacman99 @petabyteboy
@piegamesde @nbp @plabadens @polygon @polykernel @primeos @princemachiavelli
@prusnak @psibi @pstn @putchar @alyssais @raboof @ralith

rnhmjoj commented:
Ping:
@rissson @Rizary @rmcgibbo @roberth @roosemberth @ryantm
@s1341 @samueldr @sdier @stigtsp @siraben @solson @spacekookie
@spencerjanssen @srhb @stephank @superherointj @symphorien @Synthetica9
@tadfisher @taktoa @talyz @teto @tfc @thefloweringash @ThibautMarty
@tilpner @tnias @tomberek @toonn @unode @vaibhavsagar @valodim @vcunat
@Xe @xfix @yayayayaka @yorickvp @yu-re-ka @yusdacra @zakame @zhaofengli

lovesegfault reviewed (approved):
ACK

ggPeti commented:
correct and ok for me

rissson commented on maintainers/maintainer-list.nix:
```diff
@@ -9343,6 +9492,7 @@
   risson = {
     name = "Marc Schmitt";
     email = "marc.schmitt@risson.space";
+    matrix = "@rissson:lama-corp.space";
```
```suggestion
    matrix = "@risson:lama-corp.space";
```

rnhmjoj replied:
Fixed, thank you.

Ma27 reviewed (approved):
My Matrix handle is correct.

mkg20001 commented:
For easy checking who's left to verify the following should work when executed in the browser console together with a jQuery inject extension

```js
const interacted = $('.author').toArray().map(e => $(e).text().trim())

const mentioned = $('.user-mention').toArray().map(e => $(e).text().trim().replace('@', ''))

const missing = mentioned.filter(u => interacted.indexOf(u) === -1)
```

dywedir commented on maintainers/maintainer-list.nix:
```diff
@@ -2993,6 +3034,7 @@
   };
   dywedir = {
     email = "dywedir@gra.red";
+    matrix = "@dywedir:matrix.org";
```
:+1:

samueldr commented:
Are those identifiers static or irrevocable? E.g. would it be possible for `@someuser:matrix.org` to move to `@foobar:matrix.org` and someone else to take control of `@foobar:matrix.org`?

And in that scenario, is there a backing identifier in Matrix used that shows `@foobar:matrix.org` is the same user as `@someuser:matrix.org`?

What I want to know is whether we need have a something like `githubId`, a `matrixId` using an actual immutable identifier.

rnhmjoj committed:
lib/tests/maintainers: add matrix option

samueldr reviewed (changes requested):
> the identities are not verified in any way, yet.

Marking as "request changes" until all identities *left* in these changes are verified in some way.

(btw, I agree in principle with the change)

06kellyjac commented on maintainers/maintainer-list.nix:
```diff
@@ -5213,6 +5298,7 @@
   };
   jk = {
     email = "hello+nixpkgs@j-k.io";
+    matrix = "@j-k:matrix.org";
```
👍

zhaofengli commented on maintainers/maintainer-list.nix:
```diff
@@ -12248,6 +12442,7 @@
   };
   zhaofengli = {
     email = "hello@zhaofeng.li";
+    matrix = "@zhaofeng:zhaofeng.li";
```
:+1:

midchildan commented on maintainers/maintainer-list.nix:
```diff
@@ -7221,6 +7336,7 @@
   };
   midchildan = {
     email = "git@midchildan.org";
+    matrix = "@midchildan:matrix.org";
```
👍

Pacman99 reviewed (approved):
My handle is correct, fine with adding it.
Great idea!

rnhmjoj commented:
> Are those identifiers static or irrevocable?

Matrix user IDs are like email addresses: unlike the display name, this is unique over the federation and immutable.

> What I want to know is whether we need have a something like githubId, a matrixId using an actual immutable identifier.

It's more like a `githubId`, in fact I wasn't unsure whether to call it `matrix` or `matrixId`.

srhb commented on maintainers/maintainer-list.nix:
```diff
@@ -10424,6 +10589,7 @@
   };
   srhb = {
     email = "sbrofeldt@gmail.com";
+    matrix = "@srhb:matrix.org";
```
That's me!

mattchrist commented on maintainers/maintainer-list.nix:
```diff
@@ -6880,6 +6991,7 @@
   };
   mattchrist = {
     email = "nixpkgs-matt@christ.systems";
+    matrix = "@mattchrist:matrix.org";
```
I'm not active on Matrix and would rather not have this listed. If/when I am it will probably be at a different address.

rnhmjoj replied:
Sure, I've removed it now. Thank you.

lschuermann commented on maintainers/maintainer-list.nix:
```diff
@@ -6516,6 +6621,7 @@
   };
   lschuermann = {
     email = "leon.git@is.currently.online";
+    matrix = "@leon:is.currently.online";
```
```suggestion
    matrix = "@leons:is.currently.online";
```

So close. :)

rnhmjoj replied:
I had to manually type some ;) Thank you.

Princemachiavelli commented on maintainers/maintainer-list.nix:
```diff
@@ -12547,6 +12745,7 @@
   princemachiavelli = {
     name = "Josh Hoffer";
     email = "jhoffer@sansorgan.es";
+    matrix = "@princemachiavelli:matrix.org";
```
Correct!

colemickens commented:
This just makes me realize how long I've been putting off setting up my own homeserver. Let's be real, this is going to be my matrix identity for a while still, so mine  looks good. Thanks for doing this! :)

L-as reviewed (approved):
LGTM.

raboof reviewed (approved):
checks out for me

symphorien commented on maintainers/maintainer-list.nix:
```diff
@@ -10680,12 +10848,14 @@
   };
   symphorien = {
     email = "symphorien_nixpkgs@xlumurb.eu";
+    matrix = "@symphorien:xlumurb.eu";
```
👍

obfusk commented on maintainers/maintainer-list.nix:
```diff
@@ -8186,6 +8315,7 @@
   };
   obfusk = {
     email = "flx@obfusk.net";
+    matrix = "@obfusk:matrix.org";
```
:+1:

joepie91 reviewed (approved):
Mine's correct!

zeri42 commented:
please add @zeri:matrix.org at line 12589 : ) (sry for github problems prevent me from proposing the change directly)

piegamesde commented on maintainers/maintainer-list.nix:
```diff
@@ -8637,6 +8770,7 @@
   piegames = {
     name = "piegames";
     email = "nix@piegames.de";
+    matrix = "@piegamesde:matrix.org";
```
```suggestion
    matrix = "@piegames:matrix.org";
```

I'd be interested in knowning how you got that handle. There are some `piegamesde` Matrix accounts out there, but not on matrix.org.

rnhmjoj commented:
> please add @zeri:matrix.org at line 12589 : )

Done, thank you!

rnhmjoj commented:
@piegamesde: Fixed, now.

> I'd be interested in knowning how you got that handle.

I was using a vim macro to copy the github username and add matrix.org,
which is the most common matrix id, by far. I suppose I looked at your
ID, didn't notice the missing "de" at the end and pressed the macro
button.

dotlambda commented on maintainers/maintainer-list.nix:
```diff
@@ -2857,6 +2896,7 @@
   };
   dotlambda = {
     email = "rschuetz17@gmail.com";
+    matrix = "@robert:funklause.de";
```
:+1:

jonringer commented on maintainers/maintainer-list.nix:
```diff
@@ -5406,6 +5494,7 @@
   };
   jonringer = {
     email = "jonringer117@gmail.com";
+    matrix = "@jonringer:matrix.org";
```
correct :+1:

jbedo commented:
I'm happy to be added too: `@jb:vk3.wtf`

dotlambda commented on maintainers/maintainer-list.nix:
```diff
@@ -19,7 +20,8 @@
     - `name` is your, preferably real, name,
-    - `email` is your maintainer email address, and
+    - `email` is your maintainer email address,
+    - `matrix` is your Matrix user ID,
```
```suggestion
    - `matrix` is your Matrix user ID, and
```

dotlambda replied:
Never mind, I overlooked that there's more bullet points below

rnhmjoj commented:
> I'm happy to be added too: ***@***.***:vk3.wtf`

Done. Thank you!

gytis-ivaskevicius commented on maintainers/maintainer-list.nix:
```diff
@@ -4254,6 +4325,7 @@
   gytis-ivaskevicius = {
     name = "Gytis Ivaskevicius";
     email = "me@gytis.io";
+    matrix = "@gytis-ivaskevicius:matrix.org";
```
Correct :+1:

tomberek reviewed (approved):
:+1:

NinjaTrappeur commented on maintainers/maintainer-list.nix:
```diff
@@ -7960,6 +8085,7 @@
   };
   ninjatrappeur = {
     email = "felix@alternativebit.fr";
+    matrix = "@ninjatrappeur:matrix.org";
```
:+1:

jfroche commented on maintainers/maintainer-list.nix:
```diff
@@ -5138,6 +5223,7 @@
   jfroche = {
     name = "Jean-François Roche";
     email = "jfroche@pyxel.be";
+    matrix = "@jfroche:matrix.org";
```
I moved to `@jfroche:matrix.pyxel.cloud`.
Thank you

rnhmjoj commented:
@jfroche: Updated, thank you.

IreneKnapp commented on maintainers/maintainer-list.nix:
```diff
@@ -4777,6 +4860,7 @@
   };
   irenes = {
     name = "Irene Knapp";
+    matrix = "@irene:matrix.org";
```
This is incorrect. It's @irenes:matrix.org.

unode reviewed (approved):
All correct on my side.

IreneKnapp commented:
Although I am personally happy to have my matrix ID added to the file, I think this should be opt-in rather than opt-out. People from marginalized backgrounds can face concrete, real danger from having it be too easy for strangers to link their various identities together. I'm in favor of the commit that adds the "matrix" option, for maintainers who choose to add their own, but I think a bulk add of this type is irresponsible, especially since the commit is publicly visible whether or not it's ultimately merged.

I recognize that this is all information that isn't difficult to guess at for anyone who joins the chat rooms and idles in them, but there is a difference between "public" and "publicized".

I recognize that unhappy feedback of the type I'm offering is seldom welcome, especially when the work has already been done, so I don't intend to start a long discussion about this. I am offering my perspective in case anyone wants to consider it. In the event that this is merged, I will try my best to emotionally detach and accept that outcome.

I wish that this had been discussed before doing it.

Less importantly: My matrix ID as currently listed is incorrect; I left a comment with the correct one on the relevant line. Please change it.

rnhmjoj commented:
> Although I am personally happy to have my matrix ID added to the file, I think this should be opt-in rather than opt-out.

It is opt-in: I am asking for permission. I won't include any id
unless it will be verified and the maintainer is ok with this.

> I recognize that this is all information that isn't difficult to guess at for anyone who joins the chat rooms and idles in them, but there is a difference between "public" and "publicized".

The information in this PR was given voluntarily by the maintainers or
collected manually by me (using just a Matrix client and vim) in a few
hours from the NixOS Matrix space. Really, it's no more private or
difficult to access than the content of this PR.

> I recognize that unhappy feedback of the type I'm offering is seldom welcome

No, I appreciate that you're concerned with the maintainers' privacy, but I
honestly think this is a bit of an overreaction.

> I wish that this had been discussed before doing it.

In fact, I did ask on the RFC 0094 discussion and they told me it was ok to start a PR without an additional RFC.

> Less importantly: My matrix ID as currently listed is incorrect

Sorry, I should have fixed it.

> 
> Less importantly: My matrix ID as currently listed is incorrect; I left a comment with the correct one on the relevant line. Please change it.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/NixOS/nixpkgs/pull/140136#issuecomment-932604193

noneucat commented:
OK

nrdxp reviewed (approved):
Great idea! You got mine right :+1:

evalexpr commented:
Fine by me

IreneKnapp commented:
All right. I'm not here to make you feel guilty or anything. We have a difference of opinion, and I hope that it's a learning experience. I just want what's best for everyone.

I do feel that it would have been better to not put your findings here on GitHub, where they are indexed by search engines, until you got consent from people, but I also understand why that would be more work in terms of record-keeping. I also understand that the need for it is debatable. This is one of many topics where people have different perspectives about what's important, depending on their background.

With privacy issues in general, it's always quite difficult for any one person to anticipate what's going to be an issue for somebody else. That being the case, things like this are going to happen, and I do understand that.

We've both said our piece, and anyway it's done now. Please consider this approved on my part.

bobby285271 commented:
I am happy to be added too: `@bobby285271:matrix.org`.

rnhmjoj commented:
> I do feel that it would have been better to not put your findings here on GitHub, where they are indexed by search engines, until you got consent from people

You're right, it would have been better to ask before. It's just that I didn't want to burden the maintainers with finding their own ID and posting it here. It could have been done by sending out private messages on Matrix or email asking to confirm.
If anything like this will ever come up again, I'll try to be more careful.

> We've both said our piece, and anyway it's done now. Please consider this approved on my part.

Thank you.

frogamic commented on maintainers/maintainer-list.nix:
```diff
@@ -3859,6 +3923,7 @@
   };
   frogamic = {
     email = "frogamic@protonmail.com";
+    matrix = "@frogamic:matrix.org";
```
Mine is correct however I don't use or check Matrix regularly so i would not publicise it.

rnhmjoj replied:
I've removed it, now. Thank you.

frogamic reviewed (changes requested):
prefer to have mine removed.

rnhmjoj commented:
@bobby285271: Added, thank you.

IreneKnapp commented:
> > I do feel that it would have been better to not put your findings here on GitHub, where they are indexed by search engines, until you got consent from people
> 
> You're right, it would have been better to ask before. It's just that I didn't want to burden the maintainers with finding their own ID and posting it here. It could have been done by sending out private messages on Matrix or email asking to confirm. If anything like this will ever come up again, I'll try to be more careful.
> 
> > We've both said our piece, and anyway it's done now. Please consider this approved on my part.
> 
> Thank you.

Likewise, thank you.

L3afMe commented:
Mine is correct.

rnhmjoj commented on maintainers/maintainer-list.nix:
```diff
@@ -10894,6 +11069,8 @@
   tfc = {
     email = "jacek@galowicz.de";
+    matrix = "@jonge:ukvly.org";
+    #matrix = "@tfc:ukvly.org";
```
@tfc so, which one is correct?

tfc replied:
it's `@jonge:ukvly.de`

rnhmjoj replied:
Ok, thank you!

endocrimes commented on maintainers/maintainer-list.nix:
```diff
@@ -3101,6 +3145,7 @@
   };
   endocrimes = {
     email = "dani@builds.terrible.systems";
+    matrix = "@danielle:fairydust.space";
```
Please remove this - I'm not an active matrix user outside of more social nix conversations right now, which fluctuates with how busy I am with other things.

It's kinda rude to advertise a way of reaching someone that they didn't volunteer themselves (regardless of whether it could be found in another way).

rnhmjoj replied:
Done.

maxeaubrey commented on maintainers/maintainer-list.nix:
```diff
@@ -6928,6 +7040,7 @@
   };
   maxeaubrey = {
     email = "maxeaubrey@gmail.com";
+    matrix = "@maxeaubrey:matrix.org";
```
Ditto on @endocrimes' comment. This should be treated as opt-in for multitudes of reasons, whether that's usefulness of information, or someone just not wanting to collate their various accounts and identities.

rnhmjoj commented:
@maxeaubrey As I said before, it is opt-in. I'm asking maintainers whether they wants to be included in the list and I won't add any contact that it's not acknowledged. I compiled this list only for your convenience and without any privileged access (I just compared the matrix ids on the NixOS rooms with the github username).

endocrimes commented:
@rnhmjoj opt-in would be maintainers providing that information, or giving you approval to find it.

You scraping and then tying together identities however was _not opt in_, with no ahead of time opt out and _nonconsensual_, then published in a way that is now _trivially findable_. That is a breach of trust and people's privacy.

happysalada commented:
I think it's an honest mistake here. He thought it might have been a problem and was told it wasn't needed to make an RFC. I think the point has been made that this is problematic, and I don't think anyone disagrees now.
Do you think it would be okay if your handle was removed from the list or do you want other actions to be taken ?
(just trying to understand how to move forward here, I think everybody's awareness has been raised for the future).

Valodim commented:
For the record and discourse management, there is relevant discussion in the now collapsed and hard to browse comments, as github does. In the interest of not going in circles too much here, everyone who wants to engage please feel encouraged to read the whole thread.

rnhmjoj commented:
All right, it appears what I've done was a bad move, so I've removed all the user IDs that haven't been confirmed yet and updated the message accordingly.
I also sent a request to the Github support asking for the removal of the history of this PR.

fzakaria commented:
I approve for my Matrix ID — thank you.

cleverca22 commented:
and my matrix ID us `@cleverca22:matrix.org`

fgaz commented:
Mine is `@fgaz:matrix.org`

nixinator commented:
@nixinator:nixos.dev

i have have consulted with my master control AI, skynet, it approves of this change. great idea!

arianvp commented:
Could we please merge this asap and make a post on discourse instead so people can add themselves on their own terms? I got a 100 emails from this thread in a day and it's a little bit overwhelming...

davidak reviewed (approved):
Change looks good

Assuming everyone added agreed to be added

rnhmjoj commented:
Added @fzakaria, @nixinator, @matthiasbeyer, @fgaz, @cleverca22 and @addict3d. Thank you!

Also, all the git commits containing the full unchecked list have been purged from Github.

samueldr reviewed (approved):
Assuming all users left approved.

rnhmjoj commented:
> Could we please merge this asap and make a post on discourse instead so people can add themselves on their own terms? I got a 100 emails from this thread in a day and it's a little bit overwhelming...

I guess so, all the ids in the list should have been verified. We could collect IDs on the discourse forum and add them in more batches, to avoid creating tons of PRs.

talyz commented:
Mine is `@talyz:matrix.org` and I'm okay with having it added :)

endocrimes commented:
@rnhmjoj I just want to say thank you for doing that, and +1 on merging so folks can add themselves at will.

rnhmjoj commented:
@talyz added. Thank you!

rnhmjoj commented:
> @rnhmjoj I just want to say thank you for doing that, and +1 on merging so folks can add themselves at will.

I'll wait a bit more for the maintainers to reply. I'll double check everything and try to merge this tomorrow.

Ericson2314 commented:
Mine is `@ericson2314:matrix.org`. Thanks for doing this!

rnhmjoj commented:
> Mine is @ericson2314:matrix.org. Thanks for doing this!

Added. Thank you!

Mic92 commented:
mine is `@mic92:nixos.dev`

s1341 commented:
mine is: `@s1341:matrix.org`

rnhmjoj commented:
@mic92 @s1341 added. Thank you!

aanderse commented:
@rnhmjoj as per our conversation on matrix:

> aanderse: for the sake of verifying the matrix identity, can you leave a comment on the PR?

Yes, I'm @aanderse:nixos.dev. Thank you!

bendlas commented:
mine is `@bendlas:matrix.org`

rnhmjoj commented:
@bendlas added. Thank you.

petabyteboy commented:
My matrix id is correct. Thanks!

hmenke commented:
Please add `@hmenke:matrix.org`

rnhmjoj commented:
@hmenke Done. Thank you!

putchar commented:
hello @rnhmjoj  you can add `@putch4r:matrix.org`
if you dont mind

rnhmjoj commented:
> hello @rnhmjoj  you can add ***@***.***:matrix.org`

Done. Thank you!

buckley310 commented:
You can add `@buckley310:matrix.org` for me.

rnhmjoj commented:
@buckley310 Sorry, I had mistakenly removed you (among with other nine maintainers). I fixed this, now.

rnhmjoj commented:
I think I'm ready to merge, now. Here are the results:

**accepted**: 130/192
<details>
06kellyjac
0x4a6f
aaronjanse
adisbladis
ajs124
artturin
asbachb
bachp
bb010g
bendlas
berberman
blitz
buckley310
cdepillabout
chvp
cleverca22
cole-h
colemickens
crtified
cyplo
dali99
dasj
davidak
dotlambda
drupol
dywedir
eliasp
emmanuelrosa
ereslibre
ericson2314
etu
evalexpr
expipiplus1
eyjhb
fabaff
felschr
fgaz
figsoda
fortuneteller2k
fzakaria
ggpeti
gytis-ivaskevicius
happysalada
hax404
hmenke
hoverbear
iammrinal0
icy-thought
immae
infinisil
ireneknapp
jfroche
joepie91
jojosch
jonas-schievink
jonringer
jtojnar
kalbasit
kevincox
kirelagin
kloenk
l3afme
l-as
lassulus
ldesgoui
legendofmiracles
lovesegfault
lschuermann
lukegb
lunik1
ma27
manveru
maralorn
matthiasbeyer
mic92
midchildan
mjlbach
mkg20001
mvnetbiz
mweinelt
ncfavier
nh2
ninjatrappeur
nixinator
nlewo
noneucat
nrdxp
numkem
obfusk
pacman99
petabyteboy
piegamesde
primeos
princemachiavelli
psibi
putchar
raboof
ralith
rissson
rmcgibbo
roberth
s1341
samueldr
sdier
siraben
solson
spencerjanssen
srhb
stephank
superherointj
supersandro2000
symphorien
taktoa
talyz
tfc
thibautmarty
tnias
tomberek
toonn
unode
vaibhavsagar
valodim
vcunat
xe
xfix
yayayayaka
yorickvp
yu-re-ka
yusdacra
zhaofengli
</details>

**refused**: 4/192
<details>
endocrimes
frogamic
mattchrist
maxeaubrey
</details>

**unconfirmed**: 58/192
<details>
aluisioasg
baughn
guillaumedesforges
abathur
aminechikhaoui
andir
appleprincess
arianvp
shados
blaggacao
cheriimoya
codyopel
dadada
davidtwco
delroth
domenkozar
edude03
edwtjo
edolstra
ekleog
elvishjerricco
emilazy
cidem
fridh
gilligan
globin
cillianderoiste
grahamc
higebu
hqurve
htr
kraem
laikq
lheckemann
misuzu
mlieberman85
mlvzk
mudrii
musfay
onny
nbp
plabadens
polygon
polykernel
prusnak
pstn
alyssais
rizary
roosemberth
ryantm
stigtsp
spacekookie
synthetica9
tadfisher
teto
thefloweringash
tilpner
zakame

</details>

**submitted**: 7
<details>
aanderse
addict3d
bobby285271
jbedo
scriptkiddi
slothofanarchy
zeri42
</details>

In total I have added 137 Matrix IDs to the list that have been either confirmed or submitted by the maintainer. I have checked that:
1. the IDs are valid
2. the list evaluates
3. the Nixpkgs library tests pass

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/nixpkgs-maintainers-matrix-ids/15350/1
